chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p><s>Variables ki upyog karte samay, hum keval unki assign karte wali jagah change par value badalni hogi.</s></p><p> </p><p>jaha jaha pe hame pi ki value chahiye waha pe ham is variable name ko upyog kar sakte hai.. </p><p> </p><p>ab agar hame kabhi pi ki precision badhni hai to keval 1 jagah karna hoga, jaha pe hamne declare kiya tha.. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>good catch.</p>